index.bs

```diff
@@ -661,35 +667,41 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
           <ol>
             <li>
               If the {{keySystem}} member of
-              <var>config.keySystemConfiguration</var> is not one of the
+              <code>config.keySystemConfiguration</code> is not one of the
```

Comment:
Here and below I changed these to "code" because "var" defined a new variable every time and output looked a bit messier IMO. LMK if this is a legit way to reference the nested variables.

Reply:
sgtm